requirements/requirements-help.in

```diff
@@ -1,11 +1,11 @@
 llama-index-embeddings-huggingface
 
-# Because sentence-transformers doesn't like >=2
-numpy<2
+# sentence-transformers 4 requires numpy 2 (indirectly through scipy 1.14 which dropped Python 3.9 support)
+numpy>=2
 
-# Mac x86 only supports 2.2.2
+# Drop Mac x86 support
```

Comment:
See also https://github.com/Aider-AI/aider/pull/3444#issuecomment-2702280870 for an ongoing discussion about keeping or dropping Mac x86 support.

Reply:
Sure, but you can't fight time 😅 torch 2.7 just came out, they dropped Mac x86 support a year ago.
If it's any relief, Apple stopped selling Intel Macs 2 years ago and will probably stop supporting them in macOS 17 next year.

Reply:
I think that this decision should be ultimately up to the project maintainer. /cc @paul-gauthier